src/renderer/components/data-settings/data-settings.js

```diff
@@ -1035,10 +1035,9 @@ export default defineComponent({
         invidiousAPICall(subscriptionsPayload).then((response) => {
           resolve(response)
         }).catch((err) => {
-          console.error(err)
```

Comment:
Why did you remove this line?

Reply:
It's already getting logged in the invidiousAPICall function